_posts/2017-04-03-about-the-rust-community-team.md

```diff
@@ -0,0 +1,71 @@
+
+
+For reasons mentioned above, the open venues are the default for everything. Those are:
+
+* [The issue tracker](https://github.com/rust-community/team/issues
+): Where we keep everything around that we have up for discussion or that ended up in a task. You will notice that this includes planning for unannounced events. We love a good reveal, but this is no good reason to keep things secret.
+* [The meeting minutes](https://github.com/rust-community/team/tree/master/meeting-minutes
+): This is where we keep all minutes of our meetings.
+* [Our GitHub organisation](https://github.com/rust-community/): In general, all artifacts of our work can be found here.
+* [Our IRC channel](): we meet in #rust-community at irc.mozilla.org. We always include a simple access link in our announcements so that you don't have to be well-versed in IRC to participate. The IRC channel is public and logged.
+* [Our website](http://community.rs): here, you find general information about us, but also material about Rust in general.
+
+The community team meets once a week, for exactly one hour. Our bug tracker will always contain an issue with the agenda. Attendance is encouraged, but given our volunteer work, not expected. People are encouraged to be around [4 Minutes and 55 seconds](https://www.youtube.com/watch?v=9jK-NcRmVcw) before the meeting starts. The meetings are disciplined and start on time and end on time.
+
+As a backchannel, we mostly use the Rust projects email infrastructure, mostly the [community-team@rust-lang.org](mailto:community-team@rust-lang.org) email address. The backchannel is both for people who want to discuss things with us in private, but also for team members that wish to discuss things out of the open. This is sometimes necessary, if it touches personal issues or issues far too specific to the team that open debate makes sense. Still, we prefer the open process over everything, most of our conversations can still be found in `#rust-community`.
+
+We are aware that backchannels in open communities are always a matter of trust, and the backchannel has the same rules as all other Rust venues, especially the [Code of Conduct](https://www.rust-lang.org/en-US/conduct.html).
+
+## In Rust, Community is Core
+
+In the Rust project, community work is seen as parallel and as important as other contributions. For some of our members, it is the main contribution to the project, not a side issue to their coding work. This has drawn in a lot of people that have a lot of experience.
+
+Of all the Rust project teams, the community team is probably the one where the team work happens mostly outside of the projects organisation. We talk to interested parties, library maintainers, spend our time at meetups (often our own), write documentation. This isn't just about hanging around at tables - you need to be very well-informed about all aspects of the Rust projects and its workings, as you might have to point people towards their areas of interest.
```

Comment:
(the docs team writes documentation, we write a different kind, might be worth qualifying that word there)